alacritty_config_derive/src/serde_replace.rs

```diff
@@ -98,9 +99,30 @@ fn match_arms<T>(fields: &Punctuated<Field, T>) -> TokenStream2 {
                 _ => alacritty_config::SerdeReplace::replace(&mut self.#ident, value)?,
             });
         } else {
+            // Extract all `#[config(alias = "...")]` attribute values.
+            let aliases = field
+                .attrs
+                .iter()
+                .filter(|attr| (*attr).path().is_ident("config"))
+                .filter_map(|attr| attr.parse_args::<Attr>().ok())
+                .filter(|parsed| parsed.ident.as_str() == "alias")
+                .map(|parsed| {
+                    let value = parsed
+                        .param
+                        .unwrap_or_else(|| panic!("Field \"{}\" has no alias value", ident))
+                        .value();
+
+                    if value.is_empty() {
+                        panic!("Field \"{}\" has an empty alias value", ident);
+                    }
```

Comment:
thanks for the comment. I did not know that. In addition I noticed that also the compilation message regarding the usage of multiple `#[config(flatten)]` attributes was not correctly reported: I could've used as many `#[config(flatten)]` inside a struct, but the compilation message was something weird because of the way how `replace_arms` was used directly

Reply:
Now, the correct compilation messages are reported